libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/UrlUtils.java

```diff
@@ -52,11 +55,30 @@ public static String convertUrlToPunycodeIfNeeded(String url) {
         return url;
     }
 
+    /**
+     * Remove leading double slash, and inherit protocol scheme
+     */
+    public static String removeLeadingDoubleSlash(String url, String scheme) {
```

Comment:
> For consistency, how about using the same boolean isHTTPS parameter that's used by addUrlSchemeIfNeeded instead of passing the scheme as a string?

In theory, double slash notation means "inherit from the current scheme used", so it's not limited to http or https. In reality, I'm pretty sure we'll never use it for anything else.